kernelci/data/kernelci_api.py

```diff
@@ -79,8 +79,9 @@ def submit(self, data, verbose=False):
         obj_list = []
         for path, item in data.items():
             try:
-                if 'id' in item.keys():
-                    resp = self._put(path, json.dumps(item))
+                node_id = item.get('_id')
```

Comment:
On second thought, it looks like the root cause of this issue is because the API is returning Node objects with an `_id` attribute in JSON rather than `id`. Maybe the Pydantic `.id` alias only works on the objects but is not used when serialising the data to JSON - something to investigate.